src/index.js

```diff
@@ -12,7 +12,6 @@ export * from "./api";
 export * from "./experiment";
 export * from "./session";
 export * from "./events";
-export * from "./test";
```

Comment:
TODO:
This file was specifically just for creating a mock SDK. It lived in the implementation. 
One thing to worry about here is if `SDK-client` was exporting this so other repositories could use it. That might require us to at least keep it in here for now since I don't think we can pin those other repos to a different version of SDK client.

Reply:
Ugh actually looks like it is being used: https://github.com/search?q=org%3Apaypal%20insertMockSDKScript&type=code

I'll restore that file, then.